redbot/cogs/trivia/trivia.py

```diff
@@ -402,6 +405,64 @@ async def trivia_list(self, ctx: commands.Context):
             else:
                 await ctx.send(msg)
 
+    @trivia.command(name="info")
+    async def trivia_info(self, ctx: commands.Context, category: str.lower):
+        """Get information about a trivia category."""
+        try:
+            data = self.get_trivia_list(category)
+        except FileNotFoundError:
+            return await ctx.send(
+                _(
+                    "Invalid category `{name}`. See `{prefix}trivia list` for a list of "
+                    "trivia categories."
+                ).format(name=category, prefix=ctx.clean_prefix)
+            )
+        except InvalidListError:
+            return await ctx.send(
+                _(
+                    "There was an error parsing the trivia list for the `{name}` category. It "
+                    "may be formatted incorrectly."
+                ).format(name=category)
+            )
+
+        default_config = await self.config.guild(ctx.guild).all()
+        if "CONFIG" in data:
+            config_settings = {}
+            config = data.pop("CONFIG")
+            for setting in default_config:
+                config_settings[setting] = config.get(setting, default_config[setting])
+        else:
+            config_settings = default_config
+
+        config_settings = dict(sorted(config_settings.items(), key=itemgetter(0)))
+
+        embed = discord.Embed(
+            title=f'"{category}" Category Details',
+            color=await ctx.embed_colour(),
+        )
+        embed.add_field(name=_("Author"), value=data.pop("AUTHOR", italics("Not provided.")))
```

Comment:
Empty / whitespace-only string needs to be handled because we will have an empty string passed as the field value otherwise. Furthermore, placeholder text should be translated.

Also, though I'm not sure about this, I think "Author" should be "Authors" as there can also be multiple authors in the `AUTHOR` field.
```suggestion
        embed.add_field(
            name=_("Authors"), value=data.pop("AUTHOR", "").strip() or italics(_("Not provided."))
        )
```